packages/playground/blueprints/src/lib/steps/hello-world.ts

```diff
@@ -0,0 +1,18 @@
+import { StepHandler } from '.';
+
+export type HelloWorldStep = {
+	step: 'helloWorld';
+	message: string;
+};
+
+/**
+ * Prints a message to the console.
+ * @param playground Playground client.
+ * @param message The message to print.
+ */
+export const helloWorld: StepHandler<HelloWorldStep> = async (
+	playground,
+	{ message }
```

Comment:
Nice! You may also want to play with the third argument (`progress?`), see the `install-plugin.ts` step for more details on that.